fluffycloud/node_provider.py

```diff
@@ -1,21 +1,39 @@
+'''
```

Comment:
style nit: the repo uses triple quotes throughout (please edit elsewhere in this module too?)

Reply:
Fair enough, done.